treeherder/perf/models.py

```diff
@@ -173,6 +173,20 @@ def __str__(self):
         return "{} {}".format(self.value, self.push_timestamp)
 
 
+@python_2_unicode_compatible
+class IssueTracker(models.Model):
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=255, blank=False)
+    task_base_url = models.URLField(max_length=512, null=False)
+    main_url = models.URLField(max_length=512, null=False)
```

Comment:
`main_url` isn't used at the moment - will it be soon?

Reply:
Unclear, we could probably get away with leaving it out for now